doc/antora/modules/developers/pages/contributing.adoc

```diff
@@ -90,10 +90,10 @@ diff -urN freeradiusd-version $MYSRC > ~/feature-version.patch
 === 2. Select e-mail destination
 
 If you are on the developers mailing list, send the patch there.
-<<mailto:freeradius-devel@lists.freeradius.org.adoc#,freeradius-devel@lists.freeradius.org>>
+freeradius-devel@lists.freeradius.org.adoc
```

Comment:
mailto links should be replaced with ``link:mailto:<email address>[<email address>]